internal/provider/resource_definition_resource.go

```diff
@@ -255,15 +256,17 @@ func parseResourceDefinitionResponse(ctx context.Context, driverInputSchema map[
 		data.DriverInputs.ValuesString = types.StringValue(string(b))
 	}
 
-	if data.DriverInputs != nil && data.DriverInputs.SecretRefs.IsUnknown() {
+	if data.DriverInputs != nil {
```

Comment:
I wonder if we could have catched this by creating when`driver_inputs.secret_refs` wasn't ignored  https://github.com/humanitec/terraform-provider-humanitec/blob/main/internal/provider/resource_definition_resource_test.go#L183 

Could we maybe move the cases using only `secret_refs` into another test that isn't setting this flag?

Reply:
Another case or make each case set `ImportStateVerifyIgnore` explicitly.

Reply:
Updated using different `ImportStateVerifyIgnore` per case, is this what you meant?

Reply:
Yes 👍 🙇